src/llama-hparams.h

```diff
@@ -181,10 +184,13 @@ struct llama_hparams {
 
     // dimension of the rolling state embeddings
     // corresponds to Mamba's conv_states size or RWKV's token_shift states size
-    uint32_t n_embd_k_s() const;
+    uint32_t n_embd_k_s(uint32_t il = 0) const;
```

Comment:
I initially added these with the default value of `0` to avoid needing to change code that had no chance of being in the path of a hybrid model. After this PR, I think the only place that's true is the various `rwkv` methods in `llama-graph` and `llama-model`. Would it be better to always require an argument?